template.md

```diff
@@ -0,0 +1,51 @@
+<img src="../images/Ecma_RVB-003.jpg" align="right" height="70" alt="" />
+
+## Agenda for the TBD meeting of Ecma TC39
+
+- **Host**: TBD
+- **Dates**: TBD
+- **Times**:
+  - 10:00 to 17:00 PDT on TBD
+  - 10:00 to 16:00 PDT on TBD
+- **Location**: TBD
+- **Wifi**: TBD
+- **Dinner**: TBD
+- **Contact**:
+  - Name: TBD
+  - Phone: TBD
+  - Email: TBD
+
+## Logistics
+
+TBD Host Logistics (Parking, building access, etc.)
+
+### Registration
+
+TBD
+
+### Hotels
+
+TBD Hotels nearby (optional)
+
+## Agenda items
+
+1. Opening, welcome and roll call
+  1. Opening of the meeting (Mr. Neumann)
+  1. Introduction of attendees
+  1. Host facilities, local logistics
+1. Adoption of the agenda
+1. Approval of the minutes from last meeting
+1. Report from the Ecma Secretariat (15m)
+1. ECMA262 Status Updates (15m)
+1. ECMA402 Status Updates (15m)
+1. Test262 Status Updates (15m)
+1. Timeboxed agenda items: topics are timeboxed in increments of 15 minutes up to 60 minutes. Shorter proposals come first. Additional discussion comes at the end of the meeting.
+1. Stage 0+ proposals looking to advance
```

Comment:
I don't really know what existing means so I was trying to add more rigor. Does existing simply mean it's been talked about in committee before? Had an actual agenda item before?